src/components/GenericPlaceholder.tsx

```diff
@@ -31,7 +42,10 @@ export const GenericPlaceholder = ({
   noMargins = false,
   buttonVariant,
   buttonAction,
-}: GenericPlaceholderProps | GenericPlaceholderNoButtonProps) => (
+}:
+  | GenericPlaceholderProps
+  | GenericPlaceholderNoButtonProps
+  | GenericPlaceholderNoButtonAndTitleProps) => (
   <Container className={className} $noMargins={noMargins}>
     {image}
     <Title variant="subhead">{title}</Title>
```

Comment:
If it's optional, you need to check it `{title && <Title variant="subhead">{title}</Title>}`

Reply:
Done